### PR TITLE
Deno ARM install support

### PR DESCRIPTION
**Pull request by sant123**
See https://github.com/denoland/deno/pull/22298

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -9,24 +9,33 @@ if ! command -v unzip >/dev/null; then
...
+    deno_version=$1
+fi
+
 if [ "$OS" = "Windows_NT" ]; then
 	target="x86_64-pc-windows-msvc"
 else
 	case $(uname -sm) in
 	"Darwin x86_64") target="x86_64-apple-darwin" ;;
 	"Darwin arm64") target="aarch64-apple-darwin" ;;
 	"Linux aarch64")
```
Comment (mmastrac) [inline install.sh:24]: I don't think we need to worry about a version test here -- we can just enable support.

Comment (sant123) [reply install.sh:24]: Sounds good to me 🤝

**Review by littledivy — APPROVED** [review]
LGTM

**Review by mmastrac — APPROVED** [review]
LGTM!

**Review by Bostt — COMMENTED** [review]
LGTM

**Review by OBITOONDEADO1MZ — APPROVED** [review]

**Pull request merged**